src/main/java/com/google/sps/utils/UserUtils.java

```diff
@@ -30,7 +30,7 @@
 import java.util.logging.Logger;
 
 public final class UserUtils {
-    static final String sessionIdCookieName = "SessionID";
+    public static final String sessionIDCookieName = "SessionID";
```

Comment:
As long as you're changing this name, please follow the Google Java Style Guide and rename it to `SESSION_ID_COOKIE_NAME`. (Hideous, I know.)

See https://google.github.io/styleguide/javaguide.html#s5-naming